LibationWinForms/grid/GridEntry.cs

```diff
@@ -77,6 +79,16 @@ public string DisplayTags
 
 		public GridEntry(LibraryBook libraryBook) => setLibraryBook(libraryBook);
 
+		public async Task DownloadBook()
+		{
+			if (!DownloadInProgress)
+			{
+				DownloadInProgress = true;
```

Comment:
I recommend wrapping this like, or similar:

    if (!DownloadInProgress)
    {
        try
        {
            DownloadInProgress = true;
            await BookLiberation.ProcessorAutomationController.BackupSingleBookAsync(LibraryBook);
        }
        finally
        {
            DownloadInProgress = false;
        }
    }